scripts/rmc.py

```diff
@@ -98,45 +98,44 @@ def symbol_table_to_gotoc(json_filename, cbmc_filename, verbose=False, keep_temp
     cmd = ["symtab2gb", json_filename, "--out", cbmc_filename]
     return run_cmd(cmd, label="to-gotoc", verbose=verbose)
 
+def link_c_lib(src, dst, c_lib, verbose=False, quiet=False, function="main"):
+    cmd = ["goto-cc"] + ["--function", function] + [src] + c_lib + ["-o", dst]
+    return run_cmd(cmd, label="goto-cc", verbose=verbose, quiet=quiet)
+
 def run_cbmc(cbmc_filename, cbmc_args, verbose=False, quiet=False):
     cbmc_cmd = ["cbmc"] + cbmc_args + [cbmc_filename]
     return run_cmd(cbmc_cmd, label="cbmc", output_to="stdout", verbose=verbose, quiet=quiet)
 
-def run_visualize(cbmc_filename, cbmc_args, verbose=False, quiet=False, keep_temps=False, function="main", srcdir=".", wkdir=".", outdir="."):
+def run_visualize(cbmc_filename, cbmc_args=[], verbose=False, quiet=False, keep_temps=False, srcdir=".", wkdir=".", outdir="."):
     results_filename = os.path.join(outdir, "results.xml")
     coverage_filename = os.path.join(outdir, "coverage.xml")
     property_filename = os.path.join(outdir, "property.xml")
-    temp_goto_filename = os.path.join(outdir, "temp.goto")
     if not keep_temps:
-        for filename in [results_filename, coverage_filename, property_filename, temp_goto_filename]:
+        for filename in [results_filename, coverage_filename, property_filename]:
             atexit.register(delete_file, filename)
 
-    # 1) goto-cc --function main <cbmc_filename> -o temp.goto
-    # 2) cbmc --xml-ui --trace ~/rmc/library/rmc/rmc_lib.c temp.goto > results.xml
-    # 3) cbmc --xml-ui --cover location ~/rmc/library/rmc/rmc_lib.c temp.goto > coverage.xml
-    # 4) cbmc --xml-ui --show-properties ~/rmc/library/rmc/rmc_lib.c temp.goto > property.xml
-    # 5) cbmc-viewer --result results.xml --coverage coverage.xml --property property.xml --srcdir . --goto temp.goto --reportdir report
-
-    run_goto_cc(cbmc_filename, temp_goto_filename, verbose, quiet, function=function)
+    # 1) cbmc --xml-ui --trace ~/rmc/library/rmc/rmc_lib.c <cbmc_filename> > results.xml
+    # 2) cbmc --xml-ui --cover location ~/rmc/library/rmc/rmc_lib.c <cbmc_filename> > coverage.xml
+    # 3) cbmc --xml-ui --show-properties ~/rmc/library/rmc/rmc_lib.c <cbmc_filename> > property.xml
+    # 4) cbmc-viewer --result results.xml --coverage coverage.xml --property property.xml --srcdir . --goto <cbmc_filename> --reportdir report
     
     def run_cbmc_local(cbmc_args, output_to):
-        cbmc_cmd = ["cbmc"] + cbmc_args + [temp_goto_filename]
+        cbmc_cmd = ["cbmc"] + cbmc_args + [cbmc_filename]
         return run_cmd(cbmc_cmd, label="cbmc", output_to=output_to, verbose=verbose, quiet=quiet)
 
+    if "--unwinding-assertions" in cbmc_args:
+        cbmc_args.remove("--unwinding-assertions")
+
```

Comment:
We have split into property and cover flags in #263 so this variable name is different now. I don't think you need to do this anymore.